awx/settings/defaults.py

```diff
@@ -1041,26 +1041,23 @@ def IS_TESTING(argv=None):
             'class': 'django.utils.log.AdminEmailHandler',
         },
         'tower_warnings': {
-            'level': 'WARNING',
-            'class':'logging.handlers.RotatingFileHandler',
+            'class':'awx.main.utils.handlers.RotatingProductionLogHandler',
```

Comment:
Belated comment, but I think users would appreciate a comment here for _all_ "dynamic level" handlers saying that, yes, the handler _does_ have a level, but it is set via the setting, and give the setting name.